lib/utils/constants.dart

```diff
@@ -8,3 +8,6 @@ const defaultMessageBounce = false;
 
 /// Default timeout for sending transaction
 const defaultSendTimeout = Expiration.timeout(60);
+
+/// Same as [defaultSendTimeout] but in other type
+const defaultTimeout = Duration(seconds: 60);
```

Comment:
This is a generalized name, can we make it a little specified (because it's not clear from context too)?

Reply:
-